src/constants.rs

```diff
@@ -658,6 +658,6 @@ dw!(DwOp(u8) {
     DW_OP_implicit_value = 0x9e,
     DW_OP_stack_value = 0x9f,
 
-    // GNU extensions that are supported by our expression evaluator.
+// GNU extensions that are supported by our expression evaluator.
```

Comment:
I didn't realize it was rustfmt's doing when I made that comment, although I guess I should have realized. Given that, I don't think it's a big deal.